netlify.toml

```
@@ -62,6 +62,14 @@
     from = "/:lang/guides/deploy/layer-zero"
     to = "/:lang/guides/deploy/edgio"
 
+  [[redirects]]
+    from = "/:lang/migration/0.21.0"
+    to = "/:lang/guides/upgrade-to/v2"
+
+  [[redirects]]
+    from = "/:lang/migrate"
+    to = "/:lang/guides/upgrade-to/v2"
```

If we’re keeping the old migration guide around these redirects should probably point there as any old links are probably for that content. Although happy to hear a counterargument.

```suggestion
  [[redirects]]
    from = "/:lang/migration/0.21.0"
    to = "/:lang/guides/upgrade-to/v1"

  [[redirects]]
    from = "/:lang/migrate"
    to = "/:lang/guides/upgrade-to/v1"
```